pkgs/by-name/py/pyright/package.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "pyright",
+    "displayName": "Pyright",
+    "description": "Type checker for the Python language",
+    "version": "1.1.357",
+    "license": "MIT",
+    "author": {
+        "name": "Microsoft Corporation"
+    },
+    "publisher": "Microsoft Corporation",
+    "engines": {
+        "node": ">=14.0.0"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Microsoft/pyright",
+        "directory": "packages/pyright"
+    },
+    "scripts": {
+        "build": "webpack --mode production --progress",
+        "clean": "shx rm -rf ./dist ./out README.md LICENSE.txt",
+        "prepack": "npm run clean && shx cp ../../README.md . && shx cp ../../LICENSE.txt . && npm run build",
+        "webpack": "webpack --mode development --progress"
+    },
+    "optionalDependencies": {
+        "fsevents": "~2.3.3"
+    },
+    "devDependencies": {
+        "@types/node": "^17.0.45",
+        "copy-webpack-plugin": "^11.0.0",
+        "esbuild-loader": "^3.2.0",
+        "glob": "^7.2.3",
+        "jsonc-parser": "^3.2.1",
```

Comment:
Are there any other ways to add these build deps without including updated package(-lock).json files?

Reply:
apply patch?

Reply:
implemented as recommended here https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/javascript.section.md#try-to-use-upstream-packagejson-javascript-upstream-package-json